### DISCUSSION TITLE
Different readings in xDrip+ and Librelink

### DISCUSSION THREAD

**Discussion by Wils1907** [NightscoutFoundation/xDrip#3044] [General]
Hi

Does anyone know why the readings in xDrip+ and Librelink are different? Librelink tends to say I'm having a hypo - say 3.4 mmol/l - and xDrip will be something like 5.8. This happened about an hour ago so I had something to eat and Librelink now says I'm 4.9 and level and xDrip says I'm 7.8 and dropping. 

They're never the same and can show as much as a 5 mmol/l difference with different direction arrows. 

I use Libre 2 in the UK and have the latest alpha version of xDrip.

Thank you...   Wils

**Comment by PhilTU** [comment]
Different algorithm. You can calibrate in xDrip and OOP2 gives options on how to do that.

**Reply by Wils1907** [reply]
Hi PhilTU

Thanks for your reply. I've looked through the settings in xDrip+ but couldn't work out ones affect the algorithm. In OOP2, the options are Use Service, Use Foreground Service and Use Direct Intent plus the timer duration options.

**Reply by PhilTU** [reply]
Go to xDrip+ settings, search for OOP - it's in Less common settings the options are No calibration, Calibrate based on raw, Calibrate based on glucose (experimental). The latter takes the values shown on OOP2 screen which have been processed through a partial algorithm, for example just now my OOP2 screen says 92 (5.11) but the received raw value was 1002

**Reply by Wils1907** [reply]
Thanks very much, PhilTU, I'll have a play around.
